docs/sources/reference/run.md

```diff
@@ -451,6 +451,19 @@ familiar with using LXC directly.
 > you can use `--lxc-conf` to set a container's IP address, but this will not be
 > reflected in the `/etc/hosts` file.
 
+## Logging drivers (--log-driver)
+
+It is possible to specify a different logging driver for the container than the
+daemon.
+
+### Log driver: none
+
+Disables any logging for the container.
```

Comment:
please add how this affects the functioning of `docker logs` for that container, and if it changes that, add some info to the help for `docker logs` to tell the user how to know. 

I would suggest that `docker logs` should return a message like `logging disabled` so the user knows what is happening and why.